omnibus-sepolia.toml

```diff
@@ -0,0 +1,64 @@
+name = "synthetix-omnibus"
+version = "4"
+description = "Includes the full synthetix system with configurations applied"
+include = [
+    "tomls/settings.toml",
+    "tomls/ccip-testnet.toml",
+    "tomls/core.toml",
+    "tomls/permissions.toml",
+    "tomls/oracles/eth.toml",
+    "tomls/collaterals/weth.toml",
+    "tomls/pools/spartan-council.toml",
+    "tomls/permit-all-createPool.toml",
+    "tomls/permit-all-registerMarket.toml",
+    "tomls/permit-all-transferCrossChain.toml",
+]
+
+[setting.snx_package]
+defaultValue = "synthetix:3.3.5"
```

Comment:
shouldn't we update to the latest?

Reply:
no it should match mainnet I think to start?

Reply:
Yeah. This is just a revert of deleted deployment. Did not realise that at the time